adding_species_manuscript.md

```diff
@@ -74,6 +76,30 @@ Outline
         - but so far stdpopsim has been mainly restricted to well-characterised model organisms, limiting utility for non-model organisms or model organisms in the early stages of development
     - many potential users want to simulate their study organism, or develop inference methods for non-model organisms
         -   feedback from 2020/2021 stdpopsim workshops emphasized the need for simulations of a wide variety of model and non-model species
+        -->
+
+Dramatic reductions in sequencing costs are enabling unprecedented genomic data  to be collected for a huge variety of species @Ellegren2014.
```

Comment:
Suggest including some of the big sequencing efforts, for instance: Dramatic improvements in long-read sequencing technology and concomitant reductions in sequencing costs are enabling unprecedented genomic data to be collected for a huge variety of species @Ellegren2014, with ongoing efforts to systematically sequence a wide variety of species (eg. Vertebrate Genomes Project @Rhie2021, 10KP (10,000 Plants) Genome Project `@`Cheng2018).

Reply:
Agreed. (Although these projects aren't doing population-level sequencing, but will enable more of it.)